bibliopixel/drivers/PiWS281X.py

```diff
@@ -42,14 +42,18 @@
     10: 0,  # Technically SPI
 }
 
+DEFAULT_STRIP_TYPE = 0x081000
```

Comment:
We need an enum of sorts so people don't need to know the hex codes for others.

Reply:
I thought I looked for this before but this time I found it - these guys:  

https://github.com/jgarff/rpi_ws281x/blob/master/ws2811.h#L45-L64

Is there a canonical documentation page for these strips?  I'd love to include it in the code....

Let me know if this is right/complete and I'll make this change.